README.md

```diff
@@ -7,34 +7,72 @@
 [Crates.io]: https://crates.io/crates/clap-verbosity-flag
 [Documentation]: https://docs.rs/clap-verbosity-flag/
 
-Easily add a `--verbose` flag to CLIs using Clap
+Easily add `--verbose` and `--quiet` flags to CLIs using [Clap](http://crates.io/crates/clap).
 
 ## Examples
 
+```shell
+cargo add clap-verbosity-flag
+```
+
 ```rust
 use clap::Parser;
 
-// ...
 #[derive(Debug, Parser)]
 struct Cli {
     #[command(flatten)]
-    verbose: clap_verbosity_flag::Verbosity,
+    verbosity: clap_verbosity_flag::Verbosity,
 }
+
+fn main() {
+    let args = Cli::parse();
+    env_logger::Builder::new()
+        .filter_level(args.verbosity.into())
+        .init();
+    // Your code here
+}
+```
+
+For [`tracing`](https://crates.io/crates/tracing) support, use the `tracing` feature:
+
+```shell
+cargo add clap-verbosity-flag --no-default-features --features tracing
 ```
 
-By default, it'll only report errors.
-- `-q` silences output
-- `-v` show warnings
-- `-vv` show info
-- `-vvv` show debug
-- `-vvvv` show trace
+```rust
+use clap::Parser;
+
+#[derive(Debug, Parser)]
+struct Cli {
+    #[command(flatten)]
+    verbosity: clap_verbosity_flag::Verbosity,
+}
+
+fn main() {
+    let args = Cli::parse();
+    tracing_subscriber::fmt()
+        .with_max_level(args.verbosity)
+        .init();
+    // Your code here
+}
+```
+
+The default verbosity level will cause `log` / `tracing` to only report errors. The flags can be
+specified multiple times to increase or decrease the verbosity level. See the [Documentation] for
+info on how to change the default verbosity level.
+
+- silence output: `-q` / `--quiet`
+- show warnings: `-v` / `--verbose`
+- show info: `-vv` / `--verbose --verbose`
+- show debug: `-vvv` / `--verbose --verbose --verbose`
+- show trace: `-vvvv` / `--verbose --verbose --verbose --verbose`
 
 ## License
 
 Licensed under either of
 
-* Apache License, Version 2.0, ([LICENSE-APACHE](LICENSE-APACHE) or <http://www.apache.org/licenses/LICENSE-2.0>)
```

Comment:
While I generally prefer `-`, `*` is used in my template (github.com/epage/_rust) and would prefer changes like this to be there.

Reply:
Got it. The rationale was that I made it consistent with the other lists in the same file (markdown lint complains when there's two different list styles in a doc). Removed for now.